bin/usmqe_alerts_logger.py

```diff
@@ -122,16 +122,25 @@ def main():
     api = TendrlApi(auth=auth, url=url)
     logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
     logger = logging.getLogger("usmqe_alerts_logger@{}".format(args.user))
+    logger.propagate = False
     formatter = TendrlFormatter(logging.BASIC_FORMAT)
     handler = logging.StreamHandler()
     handler.setFormatter(formatter)
     logger.addHandler(handler)
     last_alerts = []
     while(True):
         alerts = api.alerts()
+        # how many alerts in current alert list we have already seen last time
+        seen_count = 0
         for alert in alerts:
             if alert not in last_alerts:
                 logger.info(alert)
+            else:
+                seen_count =+ 1
+        if seen_count > 0:
+            logger.info("Old alerts seen in last check that were not "\
+                "logged again: {0}".format(
+                    seen_count))
```

Comment:
That said, we need to use the loggers properly ...

Reply:
log message should be separate from the value/object dumped used in the message, so that when the log level prevents the log message being passed further, the object doesn't have to be serialized ...